pkg/kubectl/cmd/portforward.go

```diff
@@ -49,6 +49,7 @@ type PortForwardOptions struct {
 	RESTClient    *restclient.RESTClient
 	Config        *restclient.Config
 	PodClient     coreclient.PodsGetter
+	Address       []string
```

Comment:
I don't know if this is still active or not, but it showed up in my email and since I glanced at it, surely this should be called "Interfaces", no?

Reply:
/test pull-kubernetes-integration

Reply:
As I understand, interfaces would be like `eth0` `wlp3s0` and similar, this is list of IP addresses to listen on

Reply:
I agree with @goblain, address != interface and we're dealing with addresses here. Interface can have multiple addresses, we still need to specify which exactly we want to listen on.